nano/lib/stats_enums.hpp

```diff
@@ -250,13 +252,17 @@ enum class detail : uint8_t
 
 	// backlog
 	activated,
+
+	_last // Must be the last enum
 };
 
 /** Direction of the stat. If the direction is irrelevant, use in */
 enum class dir : uint8_t
 {
 	in,
-	out
```

Comment:
What is the significance of `_last`?
Must all of our enums finish with `_last`?

Reply:
It's needed for testing the full range `ASSERT_FALSE (nano::to_string (nano::stat::detail::_last).empty ());`